source/less/global/mixins.less

```
@@ -1,15 +1,12 @@
 // fonts
-.font-300 {
-  font-family: "Open Sans", "Arial", sans-serif;
-  font-weight: 300;
+.font-config(@weight: 400; @size: 20px; @line-height: 100%;) {
```

Не замечание к исправлению:

На заметку, по дефолту значение высоты строки чаще указывают множителем, чем процентами. Также дефолтное значение лучше ставить не менее 1.2 иначе на некоторых шрифтах строки могут _слипнутся_
```suggestion
.font-config(@weight: 400; @size: 20px; @line-height: 1.2;) {
```

---

Интересное и универсальное решение для любого размера шрита, прикольно!